config/kafka-instance-types-configuration.yaml

```diff
@@ -44,7 +44,11 @@
 #   - minInSyncReplicas: Minimum number of in sync replicas
 #   - replicationFactor: Replication factor
 #   - supportedAZModes: a list of the supported AZ modes. The possible values are "single", "multi"
-#   - lifespanSeconds: The limit lifespan of the kafka instance in seconds. If not specified then the instance never expires
+#   - [optional] lifespanSeconds: The limit lifespan of the kafka instance in seconds.
+#                                 This attribute is applied at kafka creation time. Updates to this attribute
+#                                 will not impact the already existing kafka instances.
+#                                 If not specified then the instance never expires.
```

Comment:
This is not accurate in case of EVAL instances

Reply:
hmm right. This assumes the "instance never expires" phrase is in the context of lifespanSeconds.
Maybe a different wording would be:
If not specified then the instance never expires as a cause of the lifespanSeconds attribute

What do you think?

Reply:
Yeah, that would be better.

Reply:
:+1: updated